zerver/openapi/zulip.yaml

```diff
@@ -539,6 +539,78 @@ paths:
                                     ],
                                   "id": 0,
                                 }
+                            - type: object
+                              description: |
+                                Event type for messages.
+                              properties:
+                                id:
+                                  type: integer
+                                type:
+                                  type: string
+                                  enum:
+                                    - message
+                                message:
+                                  $ref: "#/components/schemas/Messages"
+                                flags:
+                                  type: array
+                                  description: |
+                                    The user's [message flags][message-flags] for the message.
+                                  items:
+                                    type: string
+                                email_notified:
+                                  type: boolean
+                                  description: |
+                                    Whether the user receiving the event has already been
+                                    notified via email.
+                                push_notified:
+                                  type: boolean
+                                  description: |
+                                    Whether the user receiving the event has already been
+                                    notified via mobile notification.
+                                stream_push_notify:
+                                  type: boolean
+                                  description: |
+                                    Whether the user receiving the event has to be notified
+                                    via mobile notification for stream message.
+                                stream_email_notify:
+                                  type: boolean
+                                  description: |
+                                    Whether the user receiving the event has to be notified
+                                    via email for stream message.
+                                wildcard_mention_notify:
+                                  type: boolean
+                                  description: |
+                                    Whether the user has to be notified due to wildcard mention.
```

Comment:
I think `push_notified` and friends don't actually appear in the API; they're internal data structures that don't make it through to the API.  So we shouldn't have them here.

(Same with `handler_id`).

Reply:
I think this is actually our tooling revealing a bug; these fields are being accidentally leaked to the API in some code paths.